packages/webview_flutter/webview_flutter_wkwebview/pigeons/web_kit.dart

```diff
@@ -735,6 +735,37 @@ abstract class WKUIDelegateFlutterApi {
     WKFrameInfoData frame,
     WKMediaCaptureTypeData type,
   );
+
+  /// Callback to Dart function `WKUIDelegate.runJavaScriptAlertPanelWithMessage`.
+  @ObjCSelector(
+    'runJavaScriptAlertPanelForDelegateWithIdentifier:message:',
+  )
+  @async
+  void runJavaScriptAlertPanelWithIdentifier(
```

Comment:
This should be `runJavaScriptAlertPanel`. The `withIdentifier` should only be added to the `@ObjCSelector` annotation above and not in the method.